cve_bin_tool/checkers/wpa_supplicant.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+#!/usr/bin/python3
```

Comment:
```suggestion
```

Per #1123 we're going to be getting rid of these, might as well just not commit the shebang line in the first place!